src/sentry/integrations/opsgenie/integration.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+# import logging
+from typing import Any, Mapping, Sequence
+
+# from django import forms
+from django.utils.translation import gettext_lazy as _
+
+# from sentry import options
+from sentry.integrations.base import (
+    FeatureDescription,
+    IntegrationFeatures,
+    IntegrationInstallation,
+    IntegrationMetadata,
+    IntegrationProvider,
+)
+
+# from sentry.models import Integration, OrganizationIntegration
+from sentry.pipeline import PipelineView
+
+DESCRIPTION = """
+Trigger alerts in Opsgenie from Sentry.
+
+Opsgenie is a cloud-based service for dev & ops teams, providing reliable
+alerts, on-call schedule management and escalations. Opsgenie integrates with
+monitoring tools & services, ensures the right people are notified.
```

Comment:
I found the description you used from [here](https://engineering.opsgenie.com/about) but they have some typos in their own blog :| The slightly more detailed explanation below has no typos
```suggestion
Opsgenie is a cloud-based service for dev & ops teams, providing reliable
alerts, on-call schedule management, and escalations. Opsgenie integrates with
monitoring tools & services and ensures that the right people are notified via email, SMS, phone calls, and iOS & Android push notifications.
```